guardian/testapp/migrations/0001_initial.py

```diff
@@ -1,106 +1,155 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-09-16 11:15
 from __future__ import unicode_literals
 
 import datetime
-
 from django.conf import settings
-import django.core.validators
-from django.db import models, migrations
+import django.contrib.auth.models
+import django.contrib.auth.validators
```

Comment:
```django.contrib.auth.validators``` was introduced in django 1.10.1 (see https://github.com/django/django/commit/526575c64150e10dd8666d1ed3f86eedd00df2ed ). It's unacceptable to drop support for django<1.10.1. Support for django 1.9 from django foundation has not ended even. Mosts builds fails.